test/unit/lookups/esri_test.rb

```diff
@@ -22,12 +22,12 @@ def test_query_for_geocode_with_token_for_storage
     query = Geocoder::Query.new("Bluffton, SC")
     lookup = Geocoder::Lookup.get(:esri)
     res = lookup.query_url(query)
+    Geocoder.configure(esri: {token: nil, for_storage: nil})
```

Comment:
This should probably go in `teardown` in case the test exits before it gets to this line.